### ISSUE TITLE
GlassBR Tolerable probability of failure should not be an input

### ISSUE THREAD

**Issue by bmaclach** [JacquesCarette/Drasil#1238]
The recent addition of the new symbol for "tolerable probability of failure" `pb_fail` in GlassBR causes an error in the generated code because `pb_fail` is considered an input but is never given a value. I think `pb_fail` should not be an input because there is already an input `pb_tol` for the "tolerable probability of breakage", which I think is just a more concrete version of `pb_fail`. Removing `pb_fail` from the `gbInputs` list should fix this issue.

If I am wrong an `pb_fail` really should be an input, then we will need to update the input files for glassBR to provide a value for this input.

**Comment by smiths** [comment]
The addition of pb_fail was discussed at:

https://github.com/JacquesCarette/Drasil/pull/1185

Some other options that came up during the discussion might offer an idea of the right approach at this time?

**Comment by bmaclach** [comment]
Yes, I think something similar to what was done for `prob_fail` could be done here for `pb_fail`. We can make it a `ConstrainedChunk` so we don't lose constraint information, but put it into the `gbProbs` list instead of the `gbInputs` list. Currently `pb_fail` is an `UncertainChunk`, so in changing it to a `ConstrainedChunk` we would lose the uncertainty information, but I wonder whether `pb_fail` should even have uncertainty. That, I'm not sure about.

**Comment by smiths** [comment]
@bmaclach this is a helpful discussion.  We don't need uncertainty information for pb_fail, since it isn't even an input.  I think we would be fine to make it a ConstrainedChunk.  The eventual goal, as mentioned by @JacquesCarette elsewhere, is to introduce types and have a probability type.  Once we have a type for values from 0 to 1, we won't need the constraint information.

**Comment by vajihehm** [comment]
@bmaclach Thank you for you comment. I do agree with you that "pb_fail" should 'nt as an input and it could be a part of "gbProbs". @smiths Thank you for your response. As you mentioned that  we don't need uncertainty information for pb_fail, So, I put this symbol as ConstrainedChunck in commit https://github.com/JacquesCarette/Drasil/pull/1255/commits/bd43fe181676dfbe7436d16d9682e9c99b4f3de6 and pull request #1255.
